ci/build_container_centos/docker_push.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Do not ever set -x here, it is a security hazard as it will place the credentials below in the
+# Travis logs.
+set -e
+
+# push the envoy image on merge to master
+want_push='false'
+for branch in "master"
+do
+   if [ "$TRAVIS_BRANCH" == "$branch" ]
+   then
+       want_push='true'
+   fi
+done
+if [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$want_push" == "true" ]
+then
+    if [[ $(git diff HEAD^ ci/build_container_centos/) ]]; then
+        echo "There are changes in the ci/build_container_centos directory"
+        echo "Updating lyft/envoy-build-centos image"
+        cd ci/build_container_centos
+        ./docker_build.sh
+        docker login -u $DOCKERHUB_USERNAME -p $DOCKERHUB_PASSWORD
```

Comment:
Please put quotes around all string variable derefs to avoid whitespace issues.

Reply:
done